week1/week1.java

```diff
@@ -0,0 +1,72 @@
+public class week1 {
+
+	//////////Global Variables//////////
+
+	//////////Primitive Data Types//////////
+	public String myName = "George Burdell";
+	public char middleInitial = 'P';
+	//This is a comment!
+	/*
+	This is a block comment...
+	By the way there are more number types like
+	byte, short, long, and float
+	but you'll run into int and double the most
+	*/
+	public int yearsInRoboJackets = 3;
+	public double myBirthday = 8.11;
+	public Boolean isRoboJacketsCool = true;
+
+	//////////Arrays//////////
+	//You can do this with any primitive data type
+	public static double[] gpas = {1.0, 2.6, 4.0, 3.6};
+	public int[] membersInRoboJackets = new int[200];
+
+
+	public static void main(String[] args) {
+		//This is the main function that runs any logic!
+		//////////Loval Variables/////////
+		int num = 0;
+		int length = 0;
+		int count = 0;
+		String str = "";
+
+
+		//////////Operators//////////
+		System.out.println((3 + 3 == 9));
+		System.out.println((2 + 2 != 5));
+		System.out.println(!true);
+		System.out.println(true || false);
+		System.out.println(true && false);
+
+		//If-ElseIf-Else
+		if (num == 0) {
+			num = 3;
+			System.out.println("I understand Loops!");
+		} else if (num > 4) {
+			num += 3;
+			System.out.println("Like John Cena, you can't see me");
+		} else {
+			num--;
+			System.out.println("Like George P. Burdell, I don't ever show up");
+		}
+
+		//////////Loops/////////
+		//For
+		for (int n = 0; n < gpas.length; n++) {
+			System.out.println("GPA: " + gpas[n]);
+		}
+
+		//While
+		while (length < gpas.length) {
+			System.out.println("Adjusted GPA: " + (gpas[length] + 2));
+			length++;
+		}
+
+		//Do-While
+		do {
```

Comment:
This is not for teaching just showing them the code we expect them to be able to write. I am sure some freshmen is going to used to only using do whiles. Will leave this in and talk about its at the meeting.